src/main/java/org/broadleafcommerce/menu/processor/MenuProcessor.java

```diff
@@ -78,8 +73,8 @@ public Map<String, Object> populateModelVariables(String tagName, Map<String, St
 
         Map<String, Object> newModelVars = new HashMap<>();
         if (menu != null) {
-            newModelVars.put(resultVar, menuService.constructMenuItemDTOsForMenu(menu));
-            extensionManager.getProxy().addAdditionalFieldsToModel(tagName, tagAttributes, newModelVars, context);
+            newModelVars.put("menuName", menuService.constructMenuItemDTOsForMenu(menu));
```

Comment:
why do we need this line ?

Reply:
guess we need something like List<MenuItemDTO>  menuItems = menuService.constructMenuItemDTOsForMenu(menu)